### DISCUSSION TITLE
[A-L] (2023/24) Foglio 13 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#281] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8de89e47-dc15-4ac9-8944-0db1d98018a2)

**Comment by Elia-Belli** [comment]
**(4.1)** Per un corollario del Teorema della Dimensione: $T$ è iniettiva $\iff rg(T)=n$
Sia $n\gt m$, poniamo per assurdo $T$ iniettiva, abbiamo come vincolo $rg(T)\leq dim\ W$, quindi $rg(T)=n\leq m$ assurdo!
$\Rightarrow T$ non può essere iniettiva

**(4.2)** Per un corollario del Teorema della Dimensione: $T$ è suriettiva $\iff rg(T)=m$
Sia $n\lt m$, poniamo per assurdo $T$ suriettiva, abbiamo come vincolo $rg(T)\leq dim\ V$, quindi $rg(T)=m\leq n$ assurdo!
$\Rightarrow T$ non può essere suriettiva

**Comment by daromacarlo** [comment]
![Immagine 2023-12-05 181602](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/c42139cb-2158-463d-8f89-325f21bc7b0c)
![Immagine 2023-12-05 181621](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/81b64c43-1664-4477-807f-9b81bfb75cc9)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale

![Screenshot 2023-12-05 202840](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8c284051-8c01-4853-96ce-8749b71eb23f)
